CODEOWNERS

```diff
@@ -1 +1 @@
-* @Kshatrix @eromanova @a13x5 @zerospiel
+* @k0rdent/k0rdent-kcm @k0rdent/k0rdent-ksm
```

Comment:
please add team-members explicitly, the current revision contains errors

we do not want to include @DinaBelova as a code owner